src/main/java/org/folio/converter/QuickMarcToRecordConverter.java

```diff
@@ -0,0 +1,117 @@
+package org.folio.converter;
+
+import static org.folio.converter.StringConstants.CONTENT;
+
+import org.codehaus.jackson.map.ObjectMapper;
+import org.folio.rest.jaxrs.model.Field;
+import org.folio.rest.jaxrs.model.QuickMarcJson;
+import org.folio.srs.model.RawRecord;
+import org.folio.srs.model.Record;
+import org.marc4j.MarcStreamWriter;
+import org.marc4j.MarcWriter;
+import org.marc4j.marc.ControlField;
+import org.marc4j.marc.DataField;
+import org.marc4j.marc.Leader;
+import org.marc4j.marc.MarcFactory;
+import org.marc4j.marc.Subfield;
+import org.marc4j.marc.impl.MarcFactoryImpl;
+import org.springframework.core.convert.converter.Converter;
+import org.springframework.stereotype.Component;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.LinkedHashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.regex.Pattern;
+
+@Component
+public class QuickMarcToRecordConverter implements Converter<QuickMarcJson, Record> {
+  private static final int ITEM008_LENGTH = 40;
+
+  private MarcFactory factory = new MarcFactoryImpl();
+
+  @Override
+  public Record convert(QuickMarcJson quickMarcJson) {
```

Comment:
As far as we will operate with ParsedRecord (https://github.com/folio-org/data-import-raml-storage/blob/master/schemas/dto/parsedRecordDto.json) it makes sense to change signature.